### PR TITLE
Armor naming semantics

### PR DESCRIPTION
**Pull request by saltprjak**
## About The Pull Request

Changes some names and descriptions that were different-language at best, and made-up at worst. Fact-checked by a resident medieval armor enthusiast on discord. No more maille, haubergeon, halbert, or plate chausses.
image

## Why It's Good For The Game

we got rid of "bervor", might as well shake the rest of the inconsistency off

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by SaintBurgher** [comment]
Personally, I'd go with plate legs but either works

File: code/modules/clothing/rogueclothes/pants.dm [hunk 1]
```diff
@@ -161,13 +161,13 @@
...
 	name = "iron chain chausses"
-	desc = "Chain maille chausses made of iron rings woven together, offering protection against cuts and stabs."
+	desc = "Chain mail chausses made of iron rings woven together, offering protection against cuts and stabs."
 	max_integrity = 200
 	armor = list("melee" = 80, "bullet" = 50, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
 
 /obj/item/clothing/under/roguetown/platelegs
-	name = "plated chausses"
-	desc = "Chausses made of plated steel, offering additional protection against blunt force."
+	name = "plated leggings"
```
Comment (ThePainkiller) [inline code/modules/clothing/rogueclothes/pants.dm:169]:
```suggestion
	name = "plated chausses"
```
Literally every leg covering armor on Kingdom Come Deliverance is named plate chausses, I'm not budging from this one, I **know** this to be correct. Plated armor for the legs was composed by tassets for the upper thigh and cuisses for the rest of the thigh. Calling them "leggings" is leaning towards fetish territory.

File: code/modules/clothing/rogueclothes/pants.dm [hunk 2]
```diff
@@ -161,13 +161,13 @@
...
+	desc = "Leggings made of plated steel, offering additional protection against blunt force."
```
Comment (ThePainkiller) [inline code/modules/clothing/rogueclothes/pants.dm:170]:
```suggestion
	desc = "Cuisses made of plated steel, offering additional protection against blunt force."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/modules/clothing/rogueclothes/pants.dm (hunk 1 above)
Comment (Cre77) [reply code/modules/clothing/rogueclothes/pants.dm]: can't be bothered to argue with you

**Review by ThePainkiller — APPROVED** [review]

**Comment by ThePainkiller** [comment]
"Mail training" sounds funny to me, like being trained on delivering letters, but still makes sense, moving from the archaic shit we previously had (maille, sherpening wheel, etc)

**Comment by NPC1314** [comment]
Absolute garbage, the resident expert needs to go back to youtube or something.
Even wiki gives sources for maille and haubergon.

**Pull request merged**